readme.md

```diff
@@ -220,11 +220,9 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 
 ## Changelog
 
-### 3.3.1 (24 August 2022)
-- Revert changes in 3.3.0
-
-### 3.3.0 (24 August 2022) 
-- Release failed
+### 3.3.0 (24 August 2022)
```

Comment:
just for reference, we need to update the release notes. We can do so after this is merged